lib/winston/config.js

```diff
@@ -15,8 +17,9 @@ config.addColors = function (colors) {
   mixin(allColors, colors);
 };
 
-config.colorize = function (level, message) {
+config.colorize = function (level, message, auto) {
```

Comment:
I don't like `auto`, I find it hard to relate to what is doing. Maybe, `overrideBehaviour`, `autoDetect` or something like that?